pkgs/tools/graphics/termimage/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchCrate
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "termimage";
+  version = "1.2.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-1FOPe466GqQfiIpsQT9DJn+FupI2vy9b4+7p31ceY6M=";
+  };
+
+  cargoHash = "sha256-Up6wvkZJ4yLrXp/2sEAv5RqGbhLOQPNHO2vEy2Vhy+E=";
+
```

Comment:
Would you mind adding the following to install its manpage?
```nix
  nativeBuildInputs = [ ronn installShellFiles ];

  postInstall = ''
    ronn --roff --organization="termimage developers" termimage.md
    installManPage termimage.1
  '';
```

Reply:
yep, nice catch